.eslintrc.js

```diff
@@ -1,51 +1,59 @@
 // const validColors = require("./src/style/colorsPalette").colorsPalette;
 // const extraFixColorsMap = require("./src/style/colorsPalette").extraFixColorsMap;
 // const assetsDepJson = require('./eslint-rules/tests/assets_deprecation.json');
+// const deprecationsJson = require('./eslint-rules/tests/component_deprecation.json');
 
 module.exports = {
-  "parser": "babel-eslint",
-  "plugins": ["react-native", "uilib"],
-  "extends": ["airbnb"],
-  "rules": {
-    "arrow-body-style": "off",
-    "class-methods-use-this": "off",
-    "consistent-return": "off",
-    "global-require": "off",
-    "max-len": [2, 130, 4, {"ignoreUrls": true}],
-    "no-nested-ternary": "off",
-    "no-else-return": "off",
-    "no-plusplus": "off",
-    "no-return-assign": "off",
-    "no-trailing-spaces": "off",
-    "no-use-before-define": "off",
-    "no-unneeded-ternary": "off",
-    "object-curly-spacing": "off",
-    "react/forbid-prop-types": "off",
-    "react/jsx-filename-extension": "off",
-    "react/jsx-space-before-closing": "off",
-    "react/jsx-tag-spacing": "off",
-    "react/prefer-stateless-function": "off",
-    "react/require-default-props": "off",
+  'parser': 'babel-eslint',
+  'plugins': ['react-native', 'uilib'],
```

Comment:
does it mean uilib rules will be applied on our src files?

Reply:
I hoped so. But it doesn't seem to work...

Reply:
Only the colors rule doesn't work because of the export vs. module.exports things